scripts/insert_into_appointments.sql

```diff
@@ -1 +1,6 @@
 -- insert appointments creating connections between doctors and patients
+INSERT INTO appointments(patient_id, doctor_id, time, date, comment)
+  VALUES ( [Patient id of 'Marsha Wilcox'], [doctor id of 'Dr. Vicky Hilton'], 10:00, 08-12-2016)
+         ([Patient id of 'Ivory Talley'], [doctor id of 'Dr. Spencer Hubbard'], 13:00 08-17-2016)
+  + Marsha Wilcox seeing Dr. Spencer Hubbard at 10am on August 12, 2016.
+  + Blake Hodge seeing Dr. Vicky Hilton at 1pm on August 17, 2016.
```

Comment:
Hard-coding values like this will work, so I won't take points off, but it probably would have been a better idea to look up the specific patients and doctors from their respective tables. Look at the solution branch for an example.